### PR TITLE
#231 - picking up where Faizal-Majeed has left off

### PR DESCRIPTION
**Pull request by aprishchepov**
Hi Andreas,

I picked up what Faizal-Majeed has requested to merge. Created a new fresh fork from your master and ported his changes - please review and approve.

Thanks,
Arthur

### CODE REVIEW

File: UnitsNet.Tests/GeneratedCode/AccelerationTestsBase.g.cs [hunk 1]
```diff
@@ -52,76 +52,76 @@ namespace UnitsNet.Tests
...
-
-// ReSharper disable VirtualMemberNeverOverriden.Global
-        protected virtual double CentimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double DecimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double KilometerPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MicrometerPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double MillimeterPerSecondSquaredTolerance { get { return 1e-5; } }
-        protected virtual double NanometerPerSecondSquaredTolerance { get { return 1e-5; } }
+    {
```
Comment (anjdreas) [inline UnitsNet.Tests/GeneratedCode/AccelerationTestsBase.g.cs]: I think maybe you have configured your git client to enforce a certain end of line, so when you called `generate-code.bat` it replaced all lines of all files with almost identical ones.

**Comment by anjdreas** [comment]
Never mind the TeamCity errors, it was an old build server and I just disabled it.

**Comment by anjdreas** [comment]
FYI, it is useful to take a look at the `Files Changed` tab at the top of the pull request page. In this case it lists 178 changed files, which is an awful lot of files, and you quickly see that most of them are not real changes.

**Comment by anjdreas** [comment]
There are still 178 files changed in this PR, I can't merge it in before this is cleaned up. 

To my knowledge, it is probably best to start fresh on top of the latest commit (edc1968cfc6cb3c0d137af0f3a10cf025131b49a) in this repo, so you get the latest `.gitattributes` file changes. Hopefully that avoids the EOL changes you got this time. 

Let me know if you need assistance, end-of-line changes can be a bitch in git sometimes..

**Comment by anjdreas** [comment]
Great, that fixed the EOL. However github still complains it cannot be merged in due to conflicts, please rebase your branch on top of latest commit, and squash commits to tidy things up.

**Comment by anjdreas** [comment]
Huh, you are right. It looks good now. Merging it in!

**Comment by anjdreas** [comment]
Nuget 3.54 is now out

**Comment by aprishchepov** [comment]
Thanks a lot!

**Pull request merged**